p2p/pb/proto_services.go

```diff
@@ -42,12 +41,8 @@ func EncodeQuaiRequest(id uint32, location common.Location, reqData interface{},
 		reqMsg.Request = &QuaiRequestMessage_WorkObjectBlock{}
 	case *types.WorkObjectHeaderView:
 		reqMsg.Request = &QuaiRequestMessage_WorkObjectHeader{}
-	case *types.Transaction:
-		reqMsg.Request = &QuaiRequestMessage_Transaction{}
 	case common.Hash:
 		reqMsg.Request = &QuaiRequestMessage_BlockHash{}
-	case trie.TrieNodeRequest:
```

Comment:
Isn't this necessary for snapsync?

Reply:
No, we need different set of messages, like 4-5 new message types, what we did is not compatible with what ethereum has